napari/layers/transforms.py

```diff
@@ -60,12 +72,67 @@ def set_pad(self, axes: Sequence[int]) -> 'Transform':
         """
         raise NotImplementedError('Cannot subset arbitrary transforms.')
 
+
+class TransformChain(ListModel, Transform):
+    def __init__(self, transforms=[]):
+        super().__init__(
+            basetype=Transform,
+            iterable=transforms,
+            lookup={str: lambda q, e: q == e.name},
+        )
+
+    def __call__(self, coords):
+        return tz.pipe(coords, *self)
+
+    def __newlike__(self, iterable):
+        return ListModel(self._basetype, iterable, self._lookup)
+
     @property
-    def inverse(self) -> 'Transform':
-        if self._inverse_func is not None:
-            return Transform(self._inverse_func, self.func)
+    def inverse(self) -> 'TransformChain':
+        """Return the inverse transform chain."""
+        return TransformChain([tf.inverse for tf in self[::-1]])
+
+    @property
+    def composite(self) -> 'Transform':
```

Comment:
`compose` and `composite` are too similarly named & will lead to confusion. I'd suggest keeping `compose` and renaming `composite`. [Vispy's chain transform](http://vispy.org/visuals.html?highlight=chaintransform#vispy.visuals.transforms.ChainTransform) uses `simplified`, maybe we could call it something similar

Reply:
I agree, this caught me out.

Reply:
will use `simplified`, great suggestion

Reply:
switched to `simplified`